tests/integration/test.replication.js

```diff
@@ -293,13 +321,21 @@ adapters.forEach(function (adapters) {
     });
 
     it('Test pull replication with many conflicts', function (done) {
+      if (testUtils.isSyncGateway()) {
+        return done();
+      }
       var remote = new PouchDB(dbs.remote);
 
-      var numRevs = 200; // repro "url too long" error with open_revs
+      var numRevs = 10; // repro "url too long" error with open_revs
```

Comment:
So the point of the 200 here is to catch a browser error where the URL is too long. Can't recall which browsers throw exactly, but I believe it's IE and Safari.